MonumentAddons.cs

```diff
@@ -2349,9 +2349,46 @@ private void CommandHelp(IPlayer player, string cmd, string[] args)
             sb.AppendLine(GetMessage(player.Id, LangEntry.HelpShow));
             sb.AppendLine(GetMessage(player.Id, LangEntry.HelpShowVanilla));
             sb.AppendLine(GetMessage(player.Id, LangEntry.HelpProfile));
+            sb.AppendLine(GetMessage(player.Id, LangEntry.HelpSkull));
             player.Reply(sb.ToString());
         }
 
+        [Command("maskull")]
+        private void CommandSkull(IPlayer player, string cmd, string[] args) 
```

Comment:
I suggest placing this method somewhere near the other entity-related commands like after `masetdir`.

Same suggestion for the help output.